correlator_driver_functions.py

```diff
@@ -90,7 +90,7 @@ def remove_data_with_time_lags_lower_than_minimum(lags: np.ndarray, corr: np.nda
         time lag is greater than or equal to min_time_lag and Time lags that are greater
         than min_time_lag
         """
-        indices = [count for count in range(0, len(lags)) if lags[count] < min_time_lag]
+        indices = [count for count in range(0, len(lags)) if lags[count] >= min_time_lag]
```

Comment:
This function is called `remove_data_with_time_lags_lower_than_minimum` which no longer seems like what this is doing - this is now removing time lags greater-or-equal to the minimum. If this logic is correct then I think we need to adjust the function name (& docstring) too. The parameter name also needs changing - it's no longer a `min_time_lag` but actually a `max_time_lag` now.

Make sure to adjust the docstrings elsewhere too - for example `get_data_as_arrays` has a docstring which states:

> `@param min_time_lag (float): The minimum time lag to include.`

which is no longer true, it's now effectively a **maximum** time lag (any measurement with lag greater than the specified value will be discarded)